alvr/server/cpp/platform/win32/VideoEncoderSW.cpp

```diff
@@ -89,7 +89,9 @@ void VideoEncoderSW::Initialize() {
 	m_codecContext->rc_buffer_size = m_codecContext->bit_rate / settings.m_refreshRate * 1.1;
 	switch (settings.m_rateControlMode) {
 		case ALVR_CBR:
-			av_dict_set(&opt, "nal-hrd", "cbr", 0);
+			if (settings.m_fillerData) {
+				av_dict_set(&opt, "nal-hrd", "cbr", 0);
+			}
```

Comment:
Will CBR still be enabled when m_fillerData is false?

Reply:
Rate control methods are not something that is toggled in FFMPEG. They are achieved by using different rate control options. If HRD is not set, upper limit will still be hard limited (by `rc_max_rate`, `rc_buffer_size` and `bit_rate`), but filler data won't be inserted.